test/Api.Test/KeyManagement/Controllers/AccountsKeyManagementControllerTests.cs

```diff
@@ -93,4 +105,46 @@ await sutProvider.GetDependency<IRegenerateUserAsymmetricKeysCommand>().Received
                 Arg.Is(orgUsers),
                 Arg.Is(accessDetails));
     }
+
```

Comment:
Would expect unit tests for failure cases. 

`_userService.GetUserByPrincipalAsync` returning null and throwing `UnauthorizedAccessException`.

`   var result = await _rotateUserAccountKeysCommand.RotateUserAccountKeysAsync(user, dataModel);` returning `result.Errors` and throwing `BadRequestException`

Reply:
Added!